docs/content/Coding Milestones/PoC2023q1/example1.md

```diff
@@ -584,7 +584,7 @@ spec:
         ports:
         - name: http
           containerPort: 80
-          hostPort: 8082
+          hostPort: 8097
```

Comment:
This stays 8082, as long as the `containerPort` in the kind Cluster stays 8082.  This is talking about the kube node port inside the kind cluster, not the host of the kind cluster.